pages/common/ansible.md

```diff
@@ -0,0 +1,27 @@
+# ansible
+
+> Manage groups of computers remotely over SSH
+
+- Execute a command on a group of hosts:
```

Comment:
Ok, then perhaps something similar to "Run the command module on a group of hosts, passing the desired command as an argument" would be more helpful to describe this in a way that provides the right mental model for new users.

Btw, this might be a good reason to start with a module that doesn't require arguments, e.g. ping, since then we'd introduce the options gradually as recommended in the guidelines. The pages should be short enough that the most common commands not being first shouldn't pose a significant inconvenience to intermediate/experienced users.